WARNINGS.md

```diff
@@ -468,6 +469,16 @@ disabled](https://github.com/bazelbuild/bazel/issues/8391).
 
 --------------------------------------------------------------------------------
 
+## <a name="native-java"></a>All Java build rules should be loaded from Starlark
+
+  * Category name: `native-java`
```

Comment:
Please mention the flag here:

      * Flag in Bazel: [`--incompatible_load_java_rules_from_bzl`](https://github.com/bazelbuild/bazel/issues/8746)

Reply:
Done.